brokenspoke_analyzer/cli/run_with.py

```diff
@@ -81,7 +81,7 @@ def compose(
 def original_bna(
     city_shp: pathlib.Path,
     pfb_osm_file: pathlib.Path,
-    state: typing.Optional[str] = None,
+    region: typing.Optional[str] = None,
```

Comment:
This does not affect functionality, but maybe change type to `common.Region` (`typing.Optional[str]`) to ensure consistency with the other `run-with` sub-commands.